src/preferences/configobject.cpp

```diff
@@ -295,33 +285,42 @@ template class ConfigObject<ConfigValue>;
 template class ConfigObject<ConfigValueKbd>;
 
 template <> template <>
-void ConfigObject<ConfigValue>::setValue(const ConfigKey& key, const QString& value) {
+void ConfigObject<ConfigValue>::setValue(
+        const ConfigKey& key, const QString& value) {
     set(key, ConfigValue(value));
 }
 
 template <> template <>
-void ConfigObject<ConfigValue>::setValue(const ConfigKey& key, const bool& value) {
+void ConfigObject<ConfigValue>::setValue(
+        const ConfigKey& key, const bool& value) {
     set(key, value ? ConfigValue("1") : ConfigValue("0"));
 }
 
 template <> template <>
-void ConfigObject<ConfigValue>::setValue(const ConfigKey& key, const int& value) {
+void ConfigObject<ConfigValue>::setValue(
+        const ConfigKey& key, const int& value) {
     set(key, ConfigValue(QString::number(value)));
 }
 
 template <> template <>
-bool ConfigObject<ConfigValue>::getValue(const ConfigKey& key,
-                                         const bool& default_value) const {
+void ConfigObject<ConfigValue>::setValue(
+        const ConfigKey& key, const double& value) {
+    set(key, ConfigValue(QString::number(value)));
+}
+
+template <> template <>
+bool ConfigObject<ConfigValue>::getValue(
+        const ConfigKey& key, const bool& default_value) const {
     const ConfigValue value = get(key);
     if (value.isNull()) {
         return default_value;
     }
-    return static_cast<bool>(value.value.toInt());
+    return value.value.toInt() > 0;
 }
 
 template <> template <>
-int ConfigObject<ConfigValue>::getValue(const ConfigKey& key,
-                                        const int& default_value) const {
+int ConfigObject<ConfigValue>::getValue(
+        const ConfigKey& key, const int& default_value) const {
```

Comment:
The QString version requires the second format option to not exceed column 80. 
I have changed the others as well because I want to have all getValue functions sharing the same line brakes.